File: includes/addContact.php
<?php
session_start();
$userID = $_SESSION['userID'];
require("dbconnect.php");
$firstName = stripslashes($_POST['firstName']);
$lastName = stripslashes($_POST['lastName']);
$address = stripslashes($_POST['address']);
$email = stripslashes($_POST['email']);
$workNumber = stripslashes($_POST['workNumber']);
$cellNumber = stripslashes($_POST['cellNumber']);
$homeNumber = stripslashes($_POST['homeNumber']);
$faxNumber = stripslashes($_POST['faxNumber']);
$company = stripslashes($_POST['company']);
$website = stripslashes($_POST['website']);
$notes = stripslashes($_POST['notes']);

$query = "INSERT INTO contacts
(userID, firstName, lastName, address, email, workNumber, cellNumber, homeNumber, faxNumber, company, website, notes)
VALUES ('$userID', '$firstName', '$lastName', '$address', '$email', '$workNumber', '$cellNumber', '$homeNumber', '$faxNumber', '$company', '$website', '$notes');";

if($con->query($query) === false){
	trigger_error('Bad SQL: ' . $query . 'ERROR: ' . $con->error, E_USER_ERROR);	
}
else{
	$last_inserted_id = $con->insert_id;
	$affected_rows = $con->affected_rows;	
}
?>

File: cgi-bin/addContact.pl
#!/usr/bin/perl

    local ($buffer, @pairs, $pair, $name, $value, %FORM);
    # Read in text
    $ENV{'REQUEST_METHOD'} =~ tr/a-z/A-Z/;
    if ($ENV{'REQUEST_METHOD'} eq "POST")
    {
        read(STDIN, $buffer, $ENV{'CONTENT_LENGTH'});
    }else {
	$buffer = $ENV{'QUERY_STRING'};
    }
    # Split information into name/value pairs
    @pairs = split(/&/, $buffer);
    foreach $pair (@pairs)
    {
	($name, $value) = split(/=/, $pair);
	$value =~ tr/+/ /;
	$value =~ s/%(..)/pack("C", hex($1))/eg;
	$FORM{$name} = $value;
    }
    $firstName = $FORM{firstName};
    $lastName  = $FORM{lastName};
	$address  = $FORM{address};
	$cellNumber  = $FORM{cellNumber};
	$company  = $FORM{company};
	$email  = $FORM{email};
	$faxNumber  = $FORM{faxNumber};
	$homeNumber  = $FORM{homeNumber};
	$notes  = $FORM{notes};
	$website = $FORM{website};
	$workNumber  = $FORM{workNumber};

print "Content-type:text/html\r\n\r\n";
$result = "$firstName $lastName $address $cellNumber $company $email $faxNumber $homeNumber $notes $website $worknumber";
print $result;


1;